eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -226,4 +244,34 @@ public int transpile(final XML input, final Path target) throws IOException {
             this.generatedDir.toPath()
         ).save();
     }
+
+    /**
+     * Remove transpiled files per EO.
+     * @param src The eo path
+     * @return Count of removed files
+     */
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    private int removeTranspiled(final String src) {
+        final Collection<Tojo> existed = this.tojos.value().select(
+            row -> row.exists(AssembleMojo.ATTR_XMIR2)
+                && row.get(AssembleMojo.ATTR_EO).equals(src)
+        );
+        final Collection<Tojo> removable = new ArrayList<>(0);
+        for (final Tojo exist : existed) {
```

Comment:
@Karina5005 why two cycles? Why do you first add strings to a list and then you through a list and delete files. Can't you just do it in one cycle?

Reply:
Ok, fixed